src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -489,6 +489,25 @@ def _get_extension_suppressions(mod_loaders):
 
         return self.command_table
 
+    def sort_command_loaders(self, command_loaders):
+        module_command_loaders = []
```

Comment:
A small optimization: skip the sorting if there is only one element:

```suggestion
        if len(command_loaders) == 1:
            return list(command_loaders)

        module_command_loaders = []
```